Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -2237,6 +2283,40 @@ public function notifyAttendeeForSendConfirmationTrueAndPlainTextEmailCallsPostP
         $this->subject->notifyAttendee($registration, $pi1);
     }
 
+    /**
+     * @test
+     */
+    public function notifyAttendeeForSendConfirmationTrueCallsRegistrationEmailHookMethodsForHtmlEmail()
+    {
+        \Tx_Oelib_ConfigurationProxy::getInstance('seminars')
+            ->setAsInteger('eMailFormatForAttendees', TestingRegistrationManager::SEND_HTML_MAIL);
+
+        $registration = $this->createRegistration();
+
+        $hook = $this->createMock(RegistrationEmail::class);
+        $hook->expects(self::once())->method('modifyAttendeeEmail')
+            ->with(self::anything(), self::anything(), 'confirmation');
+        $hook->expects(self::once())->method('modifyAttendeeEmailBodyPlainText')
+            ->with(self::anything(), $registration, 'confirmation');
+        $hook->expects(self::once())->method('modifyAttendeeEmailBodyHtml')
+            ->with(self::anything(), $registration, 'confirmation');
```

Comment:
Please let's be a bit more specific here in the assertions. I think `self::instanceOf(…)` might work. And the registration instance might also work.